### PR TITLE
Set editors_can_change_permissions to False…

### PR DESCRIPTION
**Pull request by jnm**
everywhere, by changing its default to False and setting it to False on
all existing assets and collections

WIP - WILL FIX PR MESSAGE SHORTLY

## Checklist

1. [ ] If you've added code that should be tested, add tests
2. [ ] If you've changed APIs, update (or create!) the documentation
3. [x] Ensure the tests pass
4. [ ] Make sure that your code lints and that you've followed [our coding style](https://github.com/kobotoolbox/kpi/blob/master/CONTRIBUTING.md)
5. [ ] Write a description of your work suitable for publishing on [our forum](https://community.kobotoolbox.org/tag/release-notes)
6. [ ] Mention any related issues in this repository (as #ISSUE) and in other repositories (as kobotoolbox/other#ISSUE)

### CODE REVIEW

File: kpi/models/object_permission.py [hunk 1]
```diff
@@ -443,45 +443,23 @@ def _get_effective_perms(
...
+                    content_type.pk, codename__startswith=codename_prefix)
+                for perm_pk, perm_codename in matching_permissions:
+                    if (codename is not None and
+                            perm_codename != codename
+                    ):
+                        # If the caller specified `codename`, skip anything that
+                        # doesn't match exactly. Necessary because `Asset` has
+                        # `delete_submissions` in addition to `delete_asset`
+                        continue
+                    effective_perms.add((self.owner.pk, perm_pk))
```
Comment (noliveleger) [inline kpi/models/object_permission.py:462]: I was wondering why the part of the code was removed until I looked above and saw
> # TODO: remove this flag; support for it has been removed from
> # ObjectPermissionMixin

I guess we will implement `share_asset` as an assignable permission one day?

File: kpi/migrations/0026_disable_editors_can_change_permissions.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    Asset = apps.get_model('kpi', 'Asset')  # noqa
+    Collection = apps.get_model('kpi', 'Collection')  # noqa
+    Asset.objects.update(editors_can_change_permissions=False)
+    Collection.objects.update(editors_can_change_permissions=False)
+
+
+def reverse_func(apps, schema_editor):
+    sys.stderr.write(
+        'ENABLING `editors_can_change_permissions` on all assets and '
```
Comment (noliveleger) [inline kpi/migrations/0026_disable_editors_can_change_permissions.py:21]: Did you put this in CAPS in purpose? (vs `Disabling` in the `forward_func`)

**Review by noliveleger — APPROVED** [review]

File: kpi/migrations/0026_disable_editors_can_change_permissions.py (hunk 2 above)
Comment (jnm) [reply kpi/migrations/0026_disable_editors_can_change_permissions.py:21]: Yes, exactly :)

File: kpi/models/object_permission.py (hunk 1 above)
Comment (jnm) [reply kpi/models/object_permission.py:462]: Indeed!

**Pull request merged**